update README.md (#1661)
iter8-tools/iter8: README.md

Iter8 supports the following use-cases:

1.  Progressive release with automated traffic management
2.  A/B/n testing with a client SDK and business metrics
3.  Performance testing for HTTP and gRPC endpoints

> Any Kubernetes resource type, included CRDs can be used with Iter8.

Reviewer: ```suggestion
Any Kubernetes resource type, including CRDs, can be used with Iter8.
```

## :rocket: Features

> Iter8 introduces a set of tasks which which can be composed in order to conduct tests.

Reviewer: ```suggestion
Iter8 introduces a set of tasks that are composed together to conduct various tests.
```

Reviewer: "which which"
